### DISCUSSION TITLE
JSONAPI integration

### DISCUSSION THREAD

**Discussion by gilles-bertrand** [loco-rs/loco#305] [General]
Hello, 
I'm considering using loco for a jsonapi rest api, what would be the best way you would implement this ?
In he format.rs file, in a view ?
hank you for your help

**Comment by kaplanelad** [comment]
Hey @gilles-bertrand, can you elaborate?

**Comment by gilles-bertrand** [comment]
Hey, actually i would like to create a rest api with loco that would integrate the json-api specifications.
I'm wondering what would be the best place to start. 
I relaly like json-api for the convention part, and because I use emberjs as a frontend i would love to use loco as a backend ;)
